stripe/src/main/java/com/stripe/android/model/ConfirmPaymentIntentParams.kt

```diff
@@ -256,7 +303,8 @@ data class ConfirmPaymentIntentParams internal constructor(
             savePaymentMethod: Boolean = false,
             extraParams: Map<String, Any>? = null,
             mandateId: String? = null,
-            mandateData: MandateDataParams? = null
+            mandateData: MandateDataParams? = null,
+            setupFutureUsage: SetupFutureUsage? = null
```

Comment:
document in `fun`

OR don't add to constructors and just make it public!  But then I guess we'd want to do that for every optional param..